src/lang-go.ts

```diff
@@ -826,10 +830,18 @@ export function activate(ctx: sourcegraph.ExtensionContext = DUMMY_CTX): void {
         )
         ctx.subscriptions.add(
             sourcegraph.languages.registerReferenceProvider(goFiles, {
-                provideReferences: asyncFirst(
-                    [lsif.references, lsp.references, wrapMaybe(basicCodeIntel.references)],
-                    null
-                ),
+                provideReferences: async (doc, pos) => {
```

Comment:
Am I reading this right: if LSP is enabled, we don't use LSIF at all (nor do we fall back)?

Reply:
Yes, because LSP will provide complete results, making LSIF results redundant.